doc/dependency_decisions.yml

```diff
@@ -65,3 +65,9 @@
     :why:
     :versions: []
     :when: 2025-04-14 14:25:17.811988000 Z
+- - :permit
```

Comment:
(flyby) license finder was failing because of a `BSD-zero-clause` license. since it's an extremely permissive license, it seemed fine to include here.